### DISCUSSION TITLE
IPs from servers registered to Github are regularly attacking a private server I manage

### DISCUSSION THREAD

**Discussion by jkcunningham** [community/community#132168] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I am maintaining a private, non-commercial web-server whose only legitimate traffic comes from a small circle of known users. 90% of it's actual traffic is from port scans, seo bots ignoring robots.txt, and mostly from hacker wannabes running scripts. I'm not worried about these - it is well protected. So to a large extent, it's a de facto honeypot. Analyzing some of the hacker traffic recently I discovered  an extraordinary amount of it coming from 17 IP addresses, in an address space owned by Microsoft (Azure), but registered under the host name "Github". All of these geolocate to the same location in Dublin, Ireland. 

I once tried jumping through Microsoft's hoops to report another voluminous attack from within their server farm in Eastern Washington. But they shielded themselves from reports via an absurd reporting API. And then, after going to all the trouble of setting it up, they say, 'Sorry, we don't see anything wrong. But thanks for playing'. 

I'm guessing these 17 IP addresses are server slices leased by Github which have been hacked and are being used as remote attack devices. If I was Github, I'm pretty sure I'd want to know about this, as hacked slices can not only hack outward, but laterally and upward. But I couldn't find a way to contact Github except this discussion forum. Their site claims one can submit a ticket - even with a free account - if it's security related. But trying to do so goes round in interface circles. I gave up. 

If anyone can think of a way for me to contact them, I'll try it. I paired down log extracts which document all the attack requests. There are 4392 of them over the last 39 days. The last six attack sessions seem to be running the same script (231 identical targets each time). It was a slightly smaller sequence before that. 

But if Github has no interest, I'll just drop the whole 68.218.0.0/14 netblock with iptables and forget about it.

**Comment by davevad93** [comment, marked as answer]
Hi @jkcunningham ,

I don't know the proper way to report your findings to GitHub, I can only suggest you to open a ticket [here](https://support.github.com/contact/account), there's no label for security related stuff, so select `Other` and explain your issue.

I guess it's better than nothing, I really hope you get an answer as soon as possible because it seems like a pretty serious security issue to me.

There's also the [GitHub Bug Bounty](https://bounty.github.com/) where you can submit a vulnerability, I don't know how it works but check it out in any case.

**Reply by jkcunningham** [reply]
Thank you for the link. I've reported it.

**Reply by davevad93** [reply]
You're welcome!
